unfoldingWord/gateway-admin
# Resource Release Page

# Describe what your pull request addresses
For https://github.com/unfoldingWord/gateway-admin/issues/128

Most things should be addressed from the Epic.

## Left to do:
- Use correct OBS resources when OBS is selected
- Better error condition handling
- Artifacts such as PDFs that are currently attached to releases will not be attached. 
- A few TODO are in the code related to updating the versions of related resources in manifest.yml
- Some code cleanup could help.

## Caveats:
- I don't have access to commit to master so some steps fail

## Test Instructions

- `yarn dev`
- visit http://localhost:3000/releaseRepo?server=QA
- Select all books  and all resources
- Enter something in name and notes
- Click release
- Check log and network tab of chrome dev tools.
- Repeat to ensure newer versions numbers are used
